CHANGES

```diff
@@ -2,6 +2,7 @@
 
 - Run life cycle methods on instances from providers.
 - Log HTTP server requests even when body is not read.
+- Hide key/trust store passwords from logs.
```

Comment:
We should say "HTTP client" here, since these were already hidden for the HTTP server